diff --git a/lib/Makefile.am b/lib/Makefile.am
index 8e614605..a99c2c74 100644
--- a/lib/Makefile.am
+++ b/lib/Makefile.am
@@ -99,7 +99,11 @@ endif
 
 libsasl2.a: libsasl2.la $(SASL_STATIC_OBJS)
 	@echo adding static plugins and dependencies
+if MVS
+	$(AR) cru .libs/$@ $(SASL_STATIC_OBJS)
+else
 	$(AR) cru $@ $(SASL_STATIC_OBJS)
+endif
 	@for i in ./libsasl2.la ../common/libplugin_common.la ../sasldb/libsasldb.la ../plugins/lib*.la; do \
 	if test ! -f $$i; then continue; fi; . $$i; \
 	for j in $$dependency_libs foo; do \
